test/unit/seed-factory.js

```diff
@@ -201,10 +202,27 @@ describe("SeedFactory", () => {
             dao.address,
             admin.address,
             [seedToken.address, fundingToken.address],
+            emptyWhitelists,
             [softCap, hardCap],
             price,
-            startTime.toNumber(),
-            endTime.toNumber(),
+            [startTime.toNumber(),endTime.toNumber()],
+            [vestingDuration.toNumber(), vestingCliff.toNumber()],
+            isWhitelisted,
+            fee,
+            metadata
+          )
+        ).to.emit(seedFactory, "SeedCreated");
+      });
+      it("it creates new seed contract with whitelists", async () => {
+        await expect(
+          seedFactory.deploySeed(
+            dao.address,
+            admin.address,
+            [seedToken.address, fundingToken.address],
+            [setup.roles.root.address, setup.roles.prime.address],
+            [softCap, hardCap],
+            price,
+            [startTime.toNumber(),endTime.toNumber()],
```

Comment:
I think we don't test if the creation of the Seed was successful with batchedWhitelisting in that we check if the whitelists at the seed instance. Not sure if this extra step is needed but saw it missing

Reply:
yes, it's yet not tested

Reply:
Good point from Marvin